server/resolvers/ScenarioResult/assertionResultsResolver.js

```diff
@@ -1,6 +1,10 @@
 const { convertAssertionPriority } = require('shared');
 
-const assertionResultsResolver = (scenarioResult, { priority }) => {
+const assertionResultsResolver = (
+    scenarioResult,
+    { priority },
+    context // eslint-disable-line no-unused-vars
```

Comment:
Since the context field in GraphQL has become required for the vast majority of the resolvers, I did a pass of the server to make sure that resolvers are always called with the context variable set. I think that makes the API for using a resolver a lot more consistent and clear.

For the few resolvers that don't currently need the context I made this change just to make the expectation clear that context should be passed in now to prevent issues later.

Reply:
> For the few resolvers that don't currently need the context I made this change just to make the expectation clear that context should be passed in now to prevent issues later.

Gotcha. That seems fine with me